reward/staking_manager.go

```diff
@@ -103,14 +103,29 @@ func GetStakingManager() *StakingManager {
 	return stakingManager
 }
 
-// GetStakingInfo returns a corresponding stakingInfo for a blockNum.
+// GetStakingInfo returns a stakingInfo on the staking block of the given block number.
+// Note that staking block is the block on which the associated staking information is stored and used during an interval.
 func GetStakingInfo(blockNum uint64) *StakingInfo {
+	stakingBlockNumber := params.CalcStakingBlockNumber(blockNum)
```

Comment:
@ethan-kr Actually, the parameter is removed since it is not used in the method except for logs. Please check out the comment https://github.com/klaytn/klaytn/pull/1394#discussion_r883321856.